v2/cli/init_command_test.go

```diff
@@ -0,0 +1,65 @@
+package main
+
+import (
+	"io/ioutil"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/stretchr/testify/require"
+)
+
+func TestFileExtensionForLanguage(t *testing.T) {
+	testCases := []struct {
+		name     string
+		expected string
+	}{
+		{name: "typescript", expected: "ts"},
+		{name: "TypeScript", expected: "ts"},
+		{name: "ts", expected: "ts"},
+		{name: "javascript", expected: "js"},
+		{name: "js", expected: "js"},
+		{name: "Js", expected: "js"},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			result, err := fileExtensionForLanguage(testCase.name)
+			require.NoError(t, err)
+			require.Equal(t, result, testCase.expected)
+		})
+	}
+}
+
+func TestAddLinesToFile(t *testing.T) {
+	tempDir := t.TempDir()
+	gitIgnorePath := path.Join(tempDir, ".gitignore")
+	err := addLinesToFile(
+		gitIgnorePath,
+		path.Join(tempDir, ".brigade/secrets.yaml"),
+	)
+	require.NoError(t, err)
+
+	// Verify .gitignore is created
+	fileExists, err := file.Exists(gitIgnorePath)
+	require.True(t, fileExists)
+	require.NoError(t, err)
+
+	// Verify .gitignore contains .brigade/secrets.yaml
+	verifyContents(t, gitIgnorePath, ".brigade/secrets.yaml")
+
+	// Clear .gitignore, test functionality of writing to existing .gitignore
+	err = ioutil.WriteFile(gitIgnorePath, []byte(``), 0644)
+	require.NoError(t, err)
+	err = addLinesToFile(gitIgnorePath, ".brigade/secrets.yaml")
+	require.NoError(t, err)
+	verifyContents(t, gitIgnorePath, ".brigade/secrets.yaml")
+}
+
+func verifyContents(t *testing.T, filename string, content string) {
```

Comment:
Nit: Suggest changing `filename` to `filePath` because `filename` is often misinterpreted as being only what follows the last path delimiter (e.g. `/`). If this arg is called `filePath` instead, there's less room for misinterpretation.